packages/superset-ui-plugin-chart-table/src/Table.tsx

```diff
@@ -60,16 +67,20 @@ function getCellHash(cell: Cell) {
   return `${cell.key}#${cell.value}`;
 }
 
-function getText(value: unknown) {
+function getText(value: unknown, format: TimeFormatter | NumberFormatter | undefined) {
+  let formattedValue = value;
+  if (format) {
+    formattedValue = format.format(value as any);
+  }
   if (typeof value === 'string') {
     const span = document.createElement('span');
```

Comment:
Another new trick I learn is using regex to remove all html tags. 
I think it should be faster than parsing html via `span` tag, although haven't really tested it.

```ts
const htmlTagRegex = /(<([^>]+)>)/gi;
const text = value.replace(htmlTagRegex, '');
```